Firestore/third_party/abseil-cpp/CMakeLists.txt

```diff
@@ -70,9 +72,9 @@ set(ABSL_EXCEPTIONS_FLAG "${CMAKE_CXX_EXCEPTIONS}")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FIX_MSVC} ${CMAKE_CXX_FLAGS}")
 
 list(APPEND ABSL_TEST_COMMON_LIBRARIES
-  gtest_main
-  gtest
-  gmock
+  GTest::Main
+  GTest::GTest
+  #GMock::GMock
```

Comment:
Is this line supposed to be commented out?

Reply:
It is. None of the tests that remain actually need it.